lib/ex_format/lines.ex

```diff
@@ -0,0 +1,15 @@
+defmodule ExFormat.Lines do
+  def initialize_lines_store(string) do
+  	lines = String.split(string, "\n")
+    Agent.start_link(fn -> %{} end, name: :lines)
+    for {line, i} <- Enum.with_index(lines) do
+      update_line(i + 1, String.trim(line))
+    end
```

Comment:
Agents don't care what you initialize them with so they should have absolutely no problem being initialized with a non-empty map. What exact problem are you seeing? The Agent has an empty state? Can you show me the exact code you are using?